### PR TITLE
CEPH storage backend spported

### PR DESCRIPTION
**Pull request by jimccfun**
Changes：
1.CEPH storage backend spported
2.Fixed upload bugs

Reviewer:
@wisererik @xing-yang @leonwanghui

### CODE REVIEW

File: src/app/business/dataflow/migration.component.ts [hunk 1]
```diff
@@ -137,13 +137,20 @@ export class MigrationListComponent implements OnInit {
...
                     label:value.label,
                     value:value.value
                 });
             }
         });
+
+        // Bucket migration from CEPH to HW is not supported
+        if(Consts.BUCKET_TYPE.get(this.createMigrationForm.value.srcBucket) == "ceph-s3"){
+            this.destBuckets = this.destBuckets.filter( value => {
+                return Consts.BUCKET_TYPE.get(value.label) != "hw-obs";
```
Comment (xing-yang) [inline src/app/business/dataflow/migration.component.ts]: How about Fusion Storage?  Migration between Ceph and Fusion Storage is also not tested.

File: src/assets/business/images/common/redhat.svg [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="图层_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (xing-yang) [inline src/assets/business/images/common/redhat.svg]: id is in Chinese?

File: src/app/business/dataflow/migration.component.ts (hunk 1 above)
Comment (jimccfun) [reply src/app/business/dataflow/migration.component.ts]: Has been modified.

File: src/assets/business/images/common/redhat.svg (hunk 2 above)
Comment (jimccfun) [reply src/assets/business/images/common/redhat.svg]: Logo has been replaced.

**Pull request merged**